src/main/java/whackamole/whackamole/Translator.java

```diff
@@ -258,8 +266,39 @@ private static String typesToString(Object[] types) {
         else return " '" + String.join("', '", out) + "' ";
     }
 
+    private static void loadFiles() {
+        List<String> languages = Arrays.asList("en_US", "de_DE", "nl_NL", "fr_FR", "es_ES", "ru_RU", "tr_TR", "zh_TW");
+        File langFolder = new File(Config.AppConfig.storageFolder + "/locales");
+
+        // * Folder creation
+        if (!langFolder.exists()) {
+            langFolder.mkdirs();
+            Logger.info("Langfolder created");
+        }
+
+        // * Files creation
+        try {
+            for (String language : languages) {
```

Comment:
what if we want to add a new item for translation? This will only run once to create the file and then never again